octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py

```diff
@@ -297,12 +290,9 @@ def _create_client_config(self):
         Creates cryptofeed config
         """
         self.client_config = cryptofeed_config.Config()
-        try:
-            # Disable cryptofeed log file
-            self.client_config.config['log']['filename'] = os.devnull
-            self.client_config.config['rest']['log']['filename'] = os.devnull
-        except KeyError:
-            pass
+        # Disable cryptofeed log file
+        self.client_config.config['log'] = {}
+        self.client_config.config['log']['disabled'] = True
```

Comment:
won't this hide error logs ?

Reply:
There is no logs anymore: 
![image](https://user-images.githubusercontent.com/10454670/136089411-05a744b6-2b99-4c1e-b50c-d391fce1aaef.png)